contents/blog/startup-unified.md

```diff
@@ -0,0 +1,55 @@
+
+We don’t believe software companies should be building customer-facing integrations anymore. It just takes up too many resources. We don’t host our websites on servers in a closet anymore, so why are we still building integrations? Businesses are starting to catch on to this. You see a lot of other X-as-a-service developer tools out there: Stripe for payments, Twilio for SMS, Okta for authentication. We believe integrations should also follow that model.”
+
+### How has the journey been so far?
+“Unified.to is five months old and, while you’d think we’re pretty early on, we’re actually not. We have [product-market fit](/blog/product-market-fit-game) and we’re beating incumbent competitors when we face off against them, we’re getting our processes in order and our first customers. It’s quite exciting.
+
+We were able to progress so quickly because we really understood the problem we were trying to solve. We’re our own target customer in a lot of ways, but we also went out and did a lot of interviews with users. We spoke to about 30 organizations and that led to a lot of learnings that we didn't think about. 
+
+### What sort of insights did you get from your customers?
+“One was regarding our pricing model. The value metric that you use for pricing is really quite important. It's not the actual price, because there are obvious connotations there if you’re priced too high or too low, but the actual metric. We used to charge per connection, per account, but we discovered that users wanted more flexibility. So, we now charge based on how often users hit our API. 
+
+That was super important because it allowed our customers a lot of flexibility in terms of how often they synchronize customers' data, and is basically a one-to-one correspondence with how much they charge _their_ customers. If they're charging their customer a lot, then real-time synchronization is fine. If you're on a free plan or a very inexpensive plan, you probably don't want to do that. You'll probably want to do a daily synchronization, so that was really key.
+
+The second thing we learned from our user interviews is that there was a specific need from HR software companies. We initially thought we’d focus on sales-oriented software companies, but nearly all the customers coming in were HR software teams. So, we tweaked the roadmap to reflect their interest.”
+
+### Looking forward, what are you most excited about?
+“I'm a big data nerd and I geek out most on seeing what users are doing on my platform. I insist our teams don’t build anything they can’t measure, and that’s where PostHog comes in. I’m really excited to see the funnels, customer journeys, and understand how they change as we built. Then you can do a bunch of analysis and understand how your product _is_ being used instead of _assuming_ how it’s used.
```

Comment:
as we grow.

Reply:
```suggestion
“I'm a big data nerd and I geek out most on seeing what users are doing on my platform. I insist our teams don’t build anything they can’t measure, and that’s where PostHog comes in. I’m really excited to see the funnels, customer journeys, and understand how they change as we grow. Then you can do a bunch of analysis and understand how your product _is_ being used instead of _assuming_ how it’s used.
```